source/guides/core-concepts/variables-and-aliases.md

```diff
@@ -58,7 +58,7 @@ cy.get('button').then(($btn) => {
 })
 ```
 
-If you're familiar with native Promises the Cypress `.then()` works the same way. You can continue to nest more Cypress commands inside of the `.then()`.
+If you're familiar with {% url 'native Promises' https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Using_promises %} the Cypress `.then()` works the same way. You can continue to nest more Cypress commands inside of the `.then()`.
```

Comment:
I linked to the `Promise` docs on MDN in an effort to make the Cypress docs even more friendly and useful.